tests/Tests/Unit/CredentialsWrapperTest.php

```diff
@@ -83,7 +83,7 @@ public function buildDataWithoutExplicitKeyFile()
         $appDefaultCreds = getenv('GOOGLE_APPLICATION_CREDENTIALS');
         putenv('GOOGLE_APPLICATION_CREDENTIALS=' . __DIR__ . '/testdata/json-key-file.json');
         $scopes = ['myscope'];
-        $defaultAuthHttpHandler = HttpHandlerFactory::build();
+        $defaultAuthHttpHandler = null;
```

Comment:
It defaults to `null`, so why pass this in at all? Seems better to get rid of it and all instances of it being passed in to simplify things.